src/clr-addons/components.clr-addons.scss

```diff
@@ -25,3 +25,14 @@
     z-index: map-get($clr-layers, sidepanel) - 2;
   }
 }
+
+// Add class card-active
+.card {
+  &.card-active {
+    border-left: 0.25rem solid rgba(0, 148, 210, 1);
+
+    &.clickable:hover {
+      border-left: 0.25rem solid rgba(0, 148, 210, 1);
```

Comment:
Please use/create variables here.
If you need new variables, create their default values here.
e.g. clr-card-active-border-color: rgba(0, 148, 210, 1) !default
The !default flag indicates, that it only updates the variable if it's not already present.
That way you can customize this variable in themes (vu3 and mvap) and it will not get overwritten here.
At the same time it provides default values for clarity light+dark